src/main/java/org/alfresco/rest/requests/privateAPI/RestPrivateAPI.java

```diff
@@ -41,6 +42,7 @@
 import org.alfresco.utility.model.CustomContentModel;
 
 import io.restassured.RestAssured;
+import org.alfresco.utility.model.RepoTestModel;
```

Comment:
I don't think you have the same import order settings for this project.

Reply:
Just pushed proper import order.